### ISSUE TITLE
Error while running clustering and tree building

### ISSUE THREAD

**Issue by itigupta2429** [McGranahanLab/CONIPHER#6]
Hi Team,
I have successfully installed Conipher and while running the command via R (`conipher_run(case_id = "CRUKTOY001", prefix = "CRUK",out_dir = out_dir, input_tsv_loc = "input_table.tsv")`) I am getting the following error:

> sh: PyClone: command not found
> Error in file(con, "r") : cannot open the connection
> In addition: Warning messages:
> 1: In system(cmd) : error in running command
> 2: In file(con, "r") :
>   cannot open file 'conipher_results//Clustering///CRUKTOY001_cluster1//CRUKTOY001_S1.yaml': No such file or directory

Any possible solution to this?

**Comment by rfbrondum** [comment]
I have seen a similar issue when just installing the package into a local installation of R. When installing and running trough the conda environment on a linux machine I don't have this issue,

Running the conda environment on a windows machine gives the same error, so I expect there to be something linux specific that is needed for the clustering to run, might be the system call to PyClone.

**Comment by DenivanSchie** [comment]
I am encountering the same issue, also running it locally on a Windows installation. I have a suspicion the error arises because of the double and triple slashes. Would be great to get some feedback on this!

**Comment by itigupta2429** [comment]
Apologies for late reply.
I have ran successfully and as mentioned by @rfbrondum running through the conda environment on a linux system worked (on their dataset).
Unfortunately their seems to be some sort of issue after their latest release. Previously I was able to run complete CONIPHER pipeline (clustering and tree building) on my set of samples. But, now it gives some error in all of my samples and terminates while building the final clustering table (attached below).

> Error in pvalue(coin::independence_test(as.numeric(ccfs[order(cpn.states)]) ~  : 
>   could not find function "pvalue"
> Calls: conipher_run ... conipher_clustering -> clustering_postprocess -> copy.driven.clusterNEW
> In addition: There were 50 or more warnings (use warnings() to see the first 50)
> Execution halted
 
I am unable to fix this even after running on several different datasets. CONIPHER is running just fine on the sample dataset provided but not on my cancer sample dataset. 
Any possible solution to this what could be going wrong or how to fix this?
Thanks in advance!

**Comment by arihueb** [comment]
> I have seen a similar issue when just installing the package into a local installation of R. When installing and running trough the conda environment on a linux machine I don't have this issue,
> 
> Running the conda environment on a windows machine gives the same error, so I expect there to be something linux specific that is needed for the clustering to run, might be the system call to PyClone.

Yes, you are correct. The clustering relies on PyClone and, as this requires installation and running through a conda environment, this is necessary to successfully complete the clustering run. The conda environment and package has been tested on Linux and Mac file systems, but we haven't tested this on Windows.

If you are only interested in running the tree building, this can be done in a regular R session without first activating a conda environment.
